pkg/apis/forklift/v1beta1/doc.go

```diff
@@ -22,5 +22,81 @@ limitations under the License.
 // +groupName=forklift.konveyor.io
 package v1beta1
 
+import libcnd "github.com/konveyor/forklift-controller/pkg/lib/condition"
+
 // GlanceSource represents an image of which the source is Glance, to be used in storage mapping
 const GlanceSource = "glance"
+
+// Conditions
+const (
+	ConditionExecuting = "Executing"
```

Comment:
Another approach could be to put all condition related consts in their own subpackage, so they can be imported as `condition.Executing` rather than `api.ConditionExecuting`, if that's more appealing or easier to read.

Reply:
I like to have one place where i can see all the possible "statuses" and "conditions" , the use case I see, is a user trying to understand the "good" flow and possible "bad" flows, having a "one source of truth" for all possible flows.

Happy with current implementation, as it is a "one place" that cover all possible flows.